Jenkinsfile

```diff
@@ -33,14 +37,19 @@ node('master') {
 
         stage('Test') {
             wrap([$class: 'AnsiColorBuildWrapper', colorMapName: 'xterm']) {
-                sh './bin/behat -c build/behat.api.yml --colors -f pretty --strict'
-                sh './bin/behat -c build/behat.i18n.yml --colors -f pretty --strict'
+                timeout(time: 2, unit: 'HOURS') {
+                    if (env.WD_BROWSER_NAME == 'phantomjs') {
```

Comment:
Thanks for the comments.
Yes, this will be environment-related then.
Same reply regarding failures on Jenkins 1.